### PR TITLE
Migrate from REST to GraphQL

### LINKED ISSUE #19: [Architecture] Migrate from REST to GraphQL

### PR DESCRIPTION
**Pull request by smsmsmsmin**
Closes #19

### CODE REVIEW

File: src/resolvers/Mutation/Config.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { getConfig } from '../../resources/Config';
+import { IContext } from '../../interfaces/IContext';
+import Auth from '../../resources/Auth';
+import { ConfigModel } from '../../models';
+
+export default {
+  async setNotice(_: any, { notice }: { notice: string }, context: IContext) {
+    const updateNotice = await ConfigModel.findByIdAndUpdate(
+      '5e79c13d2f54140f4fa37bcb',
```
Comment (uhmseohun) [inline src/resolvers/Mutation/Config.ts:9]: 이건 뭔가요??

Comment (uhmseohun) [inline src/resolvers/Mutation/Config.ts:9]: `await ConfigModel.findByKey('notice')` 이렇게 하는 게 더 깔끔할 것 같은데 어떠세요??

Comment (uhmseohun) [reply src/resolvers/Mutation/Config.ts:9]: `findByKey`는 없는 것 같네요,, 괜찮으시면 메서드 추가하겠습니당

Comment (smsmsmsmin) [reply src/resolvers/Mutation/Config.ts:9]: 헉 좋아요 !! 추가해주시면 감사하겠습니다❤️

Comment (uhmseohun) [reply src/resolvers/Mutation/Config.ts:9]: 넵

Comment (uhmseohun) [reply src/resolvers/Mutation/Config.ts]: Resolved by Commit [https://github.com/dimigoin/dimigoin-back-v2/pull/57/commits/74d9180079770e07cbb4b011a2a508888a0ec3e4](https://github.com/dimigoin/dimigoin-back-v2/pull/57/commits/74d9180079770e07cbb4b011a2a508888a0ec3e4)

**Pull request merged**